include/mega/logging.h

```diff
@@ -330,6 +371,11 @@ class SimpleLogger
         logValue(lineNum);
         copyToBuffer("]", 1);
         outputBuffer();
+
+        if (!mDirectMessages.empty() && logger)
+        {
+            logger->log(nullptr, level, nullptr, "", mDirectMessages);
```

Comment:
`move(mDirectMessages)`

Reply:
good point. I'll probably use const vector & as parameters instead, to avoid vector copying